### ISSUE TITLE
[CoE Starter Kit - BUG] Not able to update solution

### ISSUE THREAD

**Issue by remzoa** [microsoft/coe-starter-kit#1089]
### Describe the issue

When I try to update my current version of Core Components from 2.75 to 2.77, I keep getting the following error message: 

Solution "Center of Excellence - Core Components" failed to import: ImportAsHolding Exception during StageAndUpgrade

This issue kept happening last time I updated to my current version, 2.75, but issue disappeared after I tried again several times. 
![CoE upgrade error](https://user-images.githubusercontent.com/73874761/131633349-9df7f11e-1878-4d3e-ab83-03cd83ccc338.PNG)

### Expected Behavior

I`m expecting that when I try to upgrade the version, I am able to upgrade the version. 

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

2.75

### What app or flow are you having the issue with?

All of them

### Steps To Reproduce

In the CoE environment. 
Go to Solutions 
Hit the Import button
Choose what to import
Select to Upgrade
Click Next
Wait 
Error appears

**Comment by Jenefer-Monroe** [comment]
When you have trouble upgrading, first note that in order to debug what happened, you will need verbose information about the failure. 
Browse to See history, find your failure, and open the information pane to see details.
In the example shown here there are missing dependencies
![image](https://user-images.githubusercontent.com/89584709/131682001-70af2ae7-9248-42dd-827c-b0628148213e.png)
![image](https://user-images.githubusercontent.com/89584709/131682405-6e91c9e8-76f7-4fcc-81ab-be1191471ef9.png)

**Comment by Jenefer-Monroe** [comment]
Then to try and fix you can first try to do an update instead of an upgrade:
![image](https://user-images.githubusercontent.com/89584709/131681136-6d9376c6-f5c0-4b3a-84ba-8b5d5bfe9540.png)

**Comment by Jenefer-Monroe** [comment]
And when all else fails, a classic overwrite will often repair the situation:

Solution > Switch to Classic (note this can take some time, may have to try twice)
![image](https://user-images.githubusercontent.com/89584709/131683084-18ee4068-873a-445c-929c-b29efcb592af.png)

Click Import (fourth from the left)
![image](https://user-images.githubusercontent.com/89584709/131683360-817b7c64-bc36-4eb5-9429-d1329001c9d3.png)

Choose File (the solution file you were trying to install) > Next > Next > 
Leave at Action=Upgrade but chose Overwrite customizations (note this is not the default) > Import

![image](https://user-images.githubusercontent.com/89584709/131683790-4e7d1f9a-62a1-4ba1-aa09-84d7160928ac.png)
![image](https://user-images.githubusercontent.com/89584709/131683907-cf80ef88-1b33-49f1-aa6c-160d825f8d7d.png)
![image](https://user-images.githubusercontent.com/89584709/131683695-fc0f19ab-a289-4854-8a6f-d6b5173b0f64.png)

**Comment by Jenefer-Monroe** [comment]
Keeping in mind that in our release notes we will tell you if there are special steps to do for this time (like there were for August)
[Release notes](https://github.com/microsoft/coe-starter-kit/releases)

**Comment by remzoa** [comment]
@Jenefer-Monroe 

Hi and thanks for the detailed description on what to try. 

The thing was, I noticed that I had 2 versions of the Core components in my environment. 
A 2.75 version, and a 2.1.70 version, and when trying to import the new version, it automatically wanted to update the 2.1.70 version instead of the 2.75, and I was not able to force it to do the update on the 2.75. 
Also, I was trying to avoid doing an Update, because the last time I did, I had to set-up the Custom Connector again. 
After fiddling around for a bit, I went into the classic version, where I was able to force it to update the 2.75 version while maintaining customizations. 

After the update was completed, the 2.1.70 version of the Core components was gone as well. Not sure what that version was, but I think that it was the cause of the issue for me. 

Everything is working and the new version is in place, so I`ll go ahead and close this one.

Thanks again for the help.
